samples/ContainerRun/Program.cs

```diff
@@ -47,6 +50,12 @@ static void Main(string[] args)
                             process.WaitForExit(5000);
                             Console.Out.WriteLine("process exited with {0}", process.ExitCode);
                         }
+                        }
+                        finally
+                        {
+                            Console.Out.WriteLine("shutting down container");
+                            container.Shutdown(Timeout.Infinite);
```

Comment:
I feel like `Shutdown(TimeSpan)` should likely have an overload that is `Shutdown()` which -> `Shutdown(Infinite)`